docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb

```diff
@@ -43,47 +43,47 @@
      "name": "stdout",
```

Comment:
<p>Should we get libdali_cusdomdummy.so here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>